### PR TITLE
Improve pages SEO & social media sharability

### PR DESCRIPTION
**Pull request by rabraghib**
- Create resolvers to fetch data
- Improve pages SEO & social media sharability (#14)

### CODE REVIEW

**Comment by rabraghib** [comment]
- SEO is actualy a 100 but netlify auto set `x-robots-tag: noindex` meta tag for deployed previews

Screenshot 2021-05-14 122519

**Comment by rabraghib** [comment]
Links social media previews:
- Twitter:
Screenshot 2021-05-14 140333
Screenshot 2021-05-14 140147

- Facebook:
Screenshot 2021-05-14 140437
Screenshot 2021-05-14 140537

**Comment by chihab** [comment]
Thanks a lot @rabraghib for the PR, I've just seen it right now. The screenshots look awesome! 
I'll share my code review asap.
Thanks again!

File: src/robots.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+User-agent: *
+Allow: /
+
+User-agent: *
+Allow: /.netlify/
```
Comment (rabraghib) [inline src/robots.txt:5]: 🤦‍♂️ I meant Disallow

**Pull request closed without merging**